src/Pages/Projects/Projects.tsx

```diff
@@ -14,17 +14,60 @@ import './Projects.module.scss';
 const Projects = () => {
 
   //! get All Projects
-  const {handleGetData} = useContext(ProjectContext)
-  const {headers} = useContext(AuthContext)
 
+  const { headers, userRole } = useContext(AuthContext)
+  const { handleGetData } = useContext(ProjectContext)
+
+
+console.log(userRole);
+const [tableData, setTableData] = useState(null)
+const [isLoading, setIsLoading] = useState(false)
+
+const getProjectsData = (pageNum:any,title:string) => {
+  setIsLoading(true)
+
```

Comment:
you can improve readability in the get method URL by doing the following:
```
const  END_POINT =  "/api/v1/Project/" + userRole==="Employee"?"employee":"manager"
return baseUrl.get(END_POINT, {
```

Reply:
or instead 
`return baseUrl.get(`/api/v1/Project/${userRole..toLowerCase()}`, {`
this seems better for me instead of having ternary operator.